app/src/androidTest/java/fr/free/nrw/commons/UITestHelper.kt

```diff
@@ -0,0 +1,63 @@
+package fr.free.nrw.commons
+
+import android.content.Context
+import androidx.test.espresso.Espresso.closeSoftKeyboard
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.NoMatchingViewException
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.matcher.ViewMatchers
+import fr.free.nrw.commons.utils.StringUtils
+import timber.log.Timber
+
+class UITestHelper {
+    companion object {
+        fun skipWelcome() {
+            try {
+                //Skip tutorial
+                onView(ViewMatchers.withId(R.id.finishTutorialButton))
+                        .perform(ViewActions.click())
+            } catch (ignored: NoMatchingViewException) {
+            }
+        }
+
+        fun loginUser() {
+            try {
+                //Perform Login
+                onView(ViewMatchers.withId(R.id.loginUsername))
+                        .perform(ViewActions.clearText(), ViewActions.typeText(getTestUsername()))
+                onView(ViewMatchers.withId(R.id.loginPassword))
+                        .perform(ViewActions.clearText(), ViewActions.typeText(getTestUserPassword()))
+                closeSoftKeyboard()
+                onView(ViewMatchers.withId(R.id.loginButton))
+                        .perform(ViewActions.click())
+            } catch (ignored: NoMatchingViewException) {
+            }
+
+        }
+
+        fun sleep(timeInMillis: Long) {
+            try {
+                Timber.d("Sleeping for %d", timeInMillis)
+                Thread.sleep(timeInMillis)
+            } catch (e: InterruptedException) {
+                e.printStackTrace()
+            }
+        }
+
+        private fun getTestUsername(): String {
+            val username = BuildConfig.TEST_USERNAME
+            if (StringUtils.isNullOrWhiteSpace(username) || username == "null") {
+                throw NotImplementedError("Configure your beta account's username")
+                //return ""
```

Comment:
This line can be removed

Reply:
I kept this line to indicate that you need to return your username. I can remove it if it is not needed.

Reply:
Sorry I meant keep in the throwing the errors, just remove the return statements (on line 51 here)